apps/jupyterhub/envs/dev/values.yaml

```
@@ -8,10 +8,39 @@ hub:
           - port: 6443
           - port: 443
   config:
+    Authenticator:
+      enable_auth_state: true
+    # Lookup URLs from
+    # https://<keycloak-url>/realms/<realm>/.well-known/openid-configuration
+    GenericOAuthenticator:
+      client_id: jupyterhub
+      client_secret: jupyterhub-client-secret
+      oauth_callback_url: https://jupyter.dev.k8tre.internal/hub/oauth_callback
+      authorize_url: https://keycloak.dev.k8tre.internal/realms/master/protocol/openid-connect/auth
+      token_url: https://keycloak.dev.k8tre.internal/realms/master/protocol/openid-connect/token
+      userdata_url: https://keycloak.dev.k8tre.internal/realms/master/protocol/openid-connect/userinfo
```

```suggestion
      token_url: http://keycloak.keycloak/realms/master/protocol/openid-connect/token
      userdata_url: http://keycloak.keycloak/realms/master/protocol/openid-connect/userinfo
```
`keycloak.keycloak` is resolvable inside the cluster, but internal connections form the hub to keycloak are currently blocked- I think the Cilium policies will need an update